### DISCUSSION TITLE
Google Login

### DISCUSSION THREAD

**Discussion by Nicole-Schwarz** [neuefische/questions#823] [Web] [answered]
Dear all,

we're trying to implement a Google-Login to our app, but facing problems (Google-Login-window opens, but is empty). 
We tried to fix it with: [https://www.npmjs.com/package/@react-oauth/google](https://www.npmjs.com/package/@react-oauth/google)

Please can anyone help us?

[https://github.com/Nesssaaa/capstone-flashcard-app/pull/70](https://github.com/Nesssaaa/capstone-flashcard-app/pull/70)

**Comment by mazzelman** [comment]
Hi Nicole,

both links are not working for me. Maybe you can provide more information about your project. Are you using Nextjs for this project? We implemented next auth and github/google to our capstone project. It is far from perfect, but you can check this out if you want: [https://github.com/marcel-epp/capstone-tripapp](https://github.com/marcel-epp/capstone-tripapp)

Maybe it helps. 😉

**Reply by Nicole-Schwarz** [reply]
I'm sorry! Now the links are working 😄 Will have a look to your code, thanks!

**Comment by Roland-Hufnagel** [comment, marked as answer]
Hi Nicole,
two links are not enough to give good help. Read here, [how to ask good questions](https://github.com/orgs/neuefische/discussions/738) 

To help you I. would need more information. What have you already implemented, what works, what not. Is the problem locally or on deployment or both? Have you tried to understand the error messages in the console? 

Apart from that I recommend to use next-auth for user login. [This repo](https://github.com/Roland-Hufnagel/Workshop-Next-Auth) may help you and give a step by step introduction.
